### PR TITLE
Addition of 'Try Again with Email' option in ticket flow process of registered user and UI changes.

### LINKED ISSUE #2442: Enhance UI for TicketFlow
**Describe the bug**

As Ticket-Flow was implemented as Eventyay V.1 , Phase - 1 and Phase - 2 are completed according to #1994 but there are many bugs created due to it .
Some are enlisted below :-
1.) When a unregistered user orders tickets or create an order, A random password is generated after this and the credentials of that users are stored in database as done in #2003,
This should be avoided as if that user wants to register for the Site , He needs to get his old password .
2.) As done in #2028  When a user enters the Email of registered user , The email is fixed to the box and the user cant edit the email after this .
3.) As stated in point 2 , The email of the user cant set empty until a refresh has been called.
4.) "You have previously registered with this email address" dialogue box generated in step 2 is encoded as a button rather than a simple message .

**Expected behavior**

Fix All such bugs and more if found .

**Desktop (please complete the following information):**
 - OS: MacOS Mojave
 - Browser: Chrome
 - Version: 73.0

**Additional context**

I would like to fix the complete ticket flow route .

### PR DESCRIPTION
**Pull request by kushthedude**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Solves #2442 

#### Changes proposed in this pull request:

- Fixed some UI and added an additional button.

Fixes #2442

### CODE REVIEW

File: app/templates/components/forms/orders/guest-order-form.hbs [hunk 1]
```diff
@@ -5,23 +5,28 @@
     </div>
   {{/if}}
   <h4 class="ui header">{{t 'Please enter your email address to continue.'}}</h4>
-  <div class="field">
-    {{input type='text' name="email" value=email  disabled=userExists placeholder=(t 'Email Address')}}
+  <div class="field" style="
+    margin-bottom: 0px;">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/orders/guest-order-form.hbs]: strictly no custom css.

File: app/templates/components/forms/orders/guest-order-form.hbs [hunk 2]
```diff
@@ -5,23 +5,28 @@
...
+    {{input type='text' name="email" value=email  enabled=userExists placeholder=(t 'Email Address')}}
   </div>
   {{#if userExists}}
-    <div class="ui text muted">{{t 'You have previously registered with this email address.'}}</div>
+    <div class="ui pointing label" style="margin-bottom: 12px;">{{t 'You have previously registered with this email address.'}}</div>
```
Comment (CosmicCoder96) [inline app/templates/components/forms/orders/guest-order-form.hbs]: strictly no custom css.

File: app/templates/components/forms/orders/guest-order-form.hbs [hunk 3]
```diff
@@ -5,23 +5,28 @@
...
-      {{input type='password' name="password" value=password}}
+      {{input type='password' name="password" value=password placeholder=(t 'Password')}}
     </div>
     <div class="ui center aligned segment basic">
       <a href="{{href-to 'reset-password'}}" class="text muted"> {{t 'Forgot your password ?'}}</a>
     </div>
     <button type="submit" class="ui blue button">
       {{t 'Sign In'}}
     </button>
+    <button type="submit" class="ui blue button" value="Refresh Page" onClick="window.location.reload()">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/orders/guest-order-form.hbs]: strictly no onclicks, no window.reload.
This logic is redundant if idea of clearing a field is fulfilled by reloading the entire app.

File: app/templates/components/forms/orders/guest-order-form.hbs [hunk 4]
```diff
@@ -5,23 +5,28 @@
...
+      Retry with New Email
```
Comment (CosmicCoder96) [inline app/templates/components/forms/orders/guest-order-form.hbs]: No translation helpers.

File: app/templates/components/forms/orders/guest-order-form.hbs [hunk 5]
```diff
@@ -5,23 +5,28 @@
...
+    </button>
+    
   {{else}}
-    <button type="submit" class="ui green button">
+    <button type="submit" class="ui green button" style="margin-top: 5px;">
```
Comment (CosmicCoder96) [inline app/templates/components/forms/orders/guest-order-form.hbs]: strictly no custom css.

**Comment by kushthedude** [comment]
Will make appropriate changes ASAP

**Comment by kushthedude** [comment]
@ritikamotwani I made the changes , but can you please tell me how do modify the CSS as i tried to create new css file in styles but it was not able to get linked ? 
Any possible help . 😅

File: app/templates/components/forms/orders/guest-order-form.hbs (hunk 3 above)
Comment (kushthedude) [reply app/templates/components/forms/orders/guest-order-form.hbs]: @CosmicCoder96 We not only have to clear a field but to we have to open the previous pop-up which only asks for email ?

**Comment by CosmicCoder96** [comment]
This functionality will be obsolete after https://github.com/fossasia/open-event-server/issues/5760 is resolved

**Pull request closed without merging**